Packs/CommonPlaybooks/Playbooks/playbook-Command-Line_Analysis.yml

```diff
@@ -1557,7 +1557,7 @@ outputs:
 - contextPath: Indicators
   description: Indicators extracted from the command line
   type: unknown
-- contextPath: commandline
+- contextPath: commandline.original
```

Comment:
we dont want to extract the indicators from the *original* coomand line.
the original command could be decoded hence, we want to extract the indicators from the decoded value or from the whole Commandline context path